### PR TITLE
[1/N][CI] Refactor accuracy test

### PR DESCRIPTION
**Pull request by Potabk**
### What this PR does / why we need it?
1. Accuracy testing no longer compares eager and graph modes; instead, it directly extracts the golden result under the graph mode configuration (the implicit purpose of this case is to verify whether modifications affect existing results)
2. Next step: finer-grained supervision of logits/sampler results
### How was this patch tested?

- vLLM version: release/v0.13.0
- vLLM main: https://github.com/vllm-project/vllm/commit/254f6b986720c92ddf97fbb1a6a6465da8e87e29

### CODE REVIEW

File: tests/e2e/singlecard/test_accuarcy.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+# @pytest.mark.parametrize("model_args", MODEL_KWARGS)
+# def test_lm_eval_accuracy_with_piecewise(model_args):
+#     run_test(model_args["pretrained"], model_args)
+
+
+@pytest.mark.parametrize("model_args", MODEL_KWARGS_FULL_FRAPH)
+def test_lm_eval_accuracy_with_full_decode_only(model_args):
+    if 'HCCL_OP_EXPANSION_MODE' in os.environ:
+        del os.environ['HCCL_OP_EXPANSION_MODE']
```
[the remark this answers is not in the document]
Comment (Potabk) [reply tests/e2e/singlecard/test_accuarcy.py]: fixed

File: tests/e2e/singlecard/test_accuarcy.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    run_test(model_args["pretrained"], model_args)
+
+
+@pytest.mark.parametrize("model_args", MODEL_KWARGS_NPUGRAPH_EX)
+def test_lm_eval_with_fullgraph_npugraph_ex(model_args):
+    if 'HCCL_OP_EXPANSION_MODE' in os.environ:
+        del os.environ['HCCL_OP_EXPANSION_MODE']
```
[the remark this answers is not in the document]
Comment (Potabk) [reply tests/e2e/singlecard/test_accuarcy.py]: fixed

**Comment by wangxiyuan** [comment]
this can be closed, right?

**Comment by Potabk** [comment]
> this can be closed, right?

This can be reuse

**Review by MengqingCao — APPROVED** [review]
